.github/workflows/CI-python-vision.yml

```diff
@@ -0,0 +1,146 @@
+name: CI Python Vision
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+    paths:
+      - "erroranalysis/**"
+      - "rai_test_utils/**"
+      - ".github/workflows/CI-python-vision.yml"
+
+jobs:
+  ci-python:
+    strategy:
+      matrix:
+        packageDirectory:
+          ["erroranalysis", "rai_test_utils"]
+        operatingSystem: [ubuntu-latest, macos-latest, windows-latest]
+        pythonVersion: ["3.8", "3.9", "3.10"]
```

Comment:
erroranalysis supports 3.6 onwards. Are we willing to drop support for them? If so, please update the setup.py file. Not testing and hoping that it works sounds dangerous to me.

@imatiach-msft FYI